src/vaadin-date-picker-mixin.html

```diff
@@ -358,6 +358,17 @@
       this._overlayContent.addEventListener('focus', () => this.focusElement._setFocused(true));
 
       this.$.overlay.addEventListener('vaadin-overlay-close', this._onVaadinOverlayClose.bind(this));
+
+      const bringToFrontListener = (e) => {
+        if (typeof this.$.overlay.bringToFront === 'function') {
+          requestAnimationFrame(() => {
+            this.$.overlay.bringToFront && this.$.overlay.bringToFront();
```

Comment:
Why the duplicate check? One on line 363 and another on 365?

Reply:
I will remove it. first line, I check it is function and second line, I check it is existed. but forgot if it is a function, it will be existed.

Reply:
done